pd/src/genesis.rs

```diff
@@ -67,6 +67,15 @@ impl FromStr for GenesisAddr {
     }
 }
 
+#[serde_as]
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub struct GenesisAppState {
+    /// Initial notes
+    pub notes: Vec<GenesisNote>,
+    /// Epoch duration in terms of blocks
+    pub epoch_duration: i64,
```

Comment:
Could we make this a `u64` and figure out how to isolate the signedness of the block heights we get, rather than propagating the signedness outwards?